packages/client/test/sim/simutils.ts

```diff
@@ -16,17 +17,37 @@ import * as fs from 'fs/promises'
 import { Level } from 'level'
 import { execSync, spawn } from 'node:child_process'
 import * as net from 'node:net'
+import qs from 'qs'
 
 import { EthereumClient } from '../../src/client'
 import { Config } from '../../src/config'
 import { LevelDB } from '../../src/execution/level'
+import { RPCManager } from '../../src/rpc'
 
 import type { Common } from '@ethereumjs/common'
 import type { TransactionType, TxData, TxOptions } from '@ethereumjs/tx'
 import type { ChildProcessWithoutNullStreams } from 'child_process'
 import type { Client } from 'jayson/promise'
 
 export const sleep = (ms: number) => new Promise((r) => setTimeout(r, ms))
+// This function switches between the native web implementation and a nodejs implemnetation
+export async function getEventSource(): Promise<typeof EventSource> {
+  // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
+  if (globalThis.EventSource) {
```

Comment:
Why disable this and not just add `gloablThis.EventSource !== undefined`?  I know it's a nit but is there a good reason to fight the linter here?